proto/tabletmanagerdata.proto

```diff
@@ -609,6 +609,7 @@ message VDiffReportOptions {
   bool only_pks = 1;
   bool debug_query = 2;
   string format = 3;
+  int64 max_report_sample_rows = 4;
```

Comment:
Since this struct is already explicitly about report related options, we could shorten the var name to: `max_sample_rows`